website/src/contribute/release-guide.md

```diff
@@ -410,7 +383,25 @@ There are 2 ways to perform this verification, either running automation script(
       ```
       ./gradlew build -PisRelease --no-parallel --scan --stacktrace --continue
       ```
-  
+
+
+#### Create release-blocking issues in JIRA
+
+The verify_release_build.sh script may include failing or flaky tests. For each of the failing tests create a JIRA with the following properties:
+
+* Issue Type: Test
```

Comment:
I think this should be "Bug". I think we just inherited the configuration from defaults, but `Test` is more-or-less a "Feature Request for Testing" according to https://issues.apache.org/jira/plugins/servlet/project-config/BEAM/issuetypes. We could have our own definitions, but since these are shared across projects probably best to make our own if we want our own.

I guess you could distinguish that it is a "Bug in a Test" versus "Bug in the product" but until it is fully diagnosed and deflaked I would actually assume a flake could be a bug in the product.

 And anyhow the distinction is too fine for my taste.

Reply:
Sounds good, changed to "Bug"